core/server/src/main/java/alluxio/master/AlluxioMaster.java

```diff
@@ -186,358 +74,49 @@ public static void main(String[] args) {
   }
 
   /**
-   * Factory for creating {@link AlluxioMaster} or {@link FaultTolerantAlluxioMaster} based on
-   * {@link Configuration}.
+   * Factory for creating {@link AlluxioMasterService}.
```

Comment:
is it better to put the factory method create to interface AlluxioMasterService, which seems more like our convention?